meerkat_frontend/src/files/pages/alerts.html

```diff
@@ -100,11 +100,18 @@
 function drawCharts( locID ){
 
 	//Whenever a new location is loaded, go back to a complete alerts table (i.e. unfiltered)
-	loadAlertTables(undefined, locID);
+  var alerts_config = Object();
+  if(config.alerts === undefined){
+    alerts_config.table_config = undefined;
+  }else{
+    alerts_config = config.alerts;
+  }
+
+	loadAlertTables(undefined, locID, alerts_config.table_config);
```

Comment:
Could probably tweak things so that these 8 lines of insertion can be put more simply in 1 line of insertion:
`loadAlertTables(undefined, locID, config.alerts_table);`
It's significantly more readable as well. I think this is another good example of that line in the "Zen of Python" poem: "flat is better than nested".  If you removed the nesting of the "table" config inside the (otherwise empty) "alert" config from this [commit](https://github.com/meerkat-code/meerkat_mad/commit/49c27315af1a34344e1229c115b22ab18d4535dd), the code above becomes much, much simpler.

Reply:
That would require to add those settings to all country configs. Is that what we want?

Reply:
I don't think it would because you have made the function resiliant to that argument being undefined.